### DISCUSSION TITLE
Welcome to LiveCharts2 Discussions!

### DISCUSSION THREAD

**Discussion by beto-rodriguez** [Live-Charts/LiveCharts2#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kMOsmane** [comment]
I'am khodor osmane, I have 22 years old, I'm from lebanon and I'm an engineering student, embeeded system and real time.
Actullay, I do a double cursus in France and I have just started my work-study program at SeaSideTech in france , so I'm here to get some help and share our experiences to be better !!! 
I have an UserInterface project and  I choosed liveChart package because I worked in it since 4 months ago to create a realtime cartesianchart like "Logic 2 Saleae" application where I have 1 million point to drawing in real time so i hope I will arive to finish my job ...
finaly, big thank to your help @beto-rodriguez .

**Comment by dlwhdvlf71** [comment]
Thank you so much for providing the LiveCharts2 library.
I'm practicing using LiveCharts2 while developing with MAUI. When will the official release be released?

**Comment by luchianov** [comment]
Thanks for such a versatile and easy to use library!
Are you folks planning to port your product to flutter/dart? 
You might discover that you can address a large segment of mobile development.
